src/benchmark.cc

```diff
@@ -193,9 +203,11 @@ void RunInThread(const benchmark::internal::Benchmark::Instance* b,
   manager->NotifyThreadComplete();
 }
 
-std::vector<BenchmarkReporter::Run> RunBenchmark(
-    const benchmark::internal::Benchmark::Instance& b,
-    std::vector<BenchmarkReporter::Run>* complexity_reports) {
+std::pair<std::vector<BenchmarkReporter::Run> /*everything*/,
```

Comment:
is it worth returning a custom struct to avoid having the comments tracking which member of the pair is which?

Reply:
Makes sense.